packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -20,7 +20,7 @@ export enum ModalMode {
 }
 
 export const toItem = (line: InboundShipmentItem | InvoiceLine): Item => ({
-  id: 'lines' in line ? line.lines[0].id : line.itemId,
+  id: 'lines' in line ? line.lines[0].itemId : line.itemId,
```

Comment:
oops, was using the invoice line ID as the item ID to lookup all of the invoice lines with the same item ID